### ISSUE TITLE
Actor/policy loss steep rise, and mean reward steep fall

### ISSUE THREAD

**Issue by EugeneLiu01** [TIGER-AI-Lab/Pixel-Reasoner#31]
Im trying single-node training, 1*8 A100
```
cd curiosity_driven_rl

export temperature=1.0
export trainver="release"
export testver="vstar"
export filter=True # filtering zero advantages
export algo=group # default for grpo
export lr=10
export MAX_PIXELS=4014080 # =[max_image_token]x28x28
export sys=vcot # system prompt version
export mode=train # [no_eval, eval_only, train]
export policy="models/PixelReasoner-WarmStart"
export rbuffer=128 # replay buffer size
export bsz=64 # global train batch size
export evalsteps=1
export mbsz=2 
export tp=1 # vllm tp, 1 for 7B
export repeat=1 # data repeat
export nepoch=3 # data epoch
export logp_bsz=1 # must be 1
export maxlen=10000 # generate_max_len
export tagname=Train

export working_dir="Pixel-Reasoner"

bash ./scripts/train_vlm_single_test2.sh
```
```
ray job submit --address="http://127.0.0.1:8265" \
--runtime-env-json="$RUNTIME_ENV_JSON" \
-- python3 -m openrlhf.cli.train_ppo_ray \
--vllm_enable_sleep \
--vllm_gpu_memory_utilization 0.85 \
--vllm_sync_backend gloo \
--pretrain $PRETRAIN_MODEL \
--save_path $SAVE_PATH \
--n_samples_per_prompt ${nsamples} \
--max_epochs 1 \
--num_episodes ${nepoch} \
--filter ${filter} \
--prompt_max_len 2048 \
--max_out_tokens ${maxtoken} \
--max_samples 100000 \
--generate_max_len ${maxlen} \
--advantage_estimator ${algo} \
--zero_stage 3 \
--controlled_shuffle ${repeat} \
--bf16 \
--actor_learning_rate ${lr}e-7 \
--rule_reward ${rule_reward} \
--temperature 1.0 \
--val_temperature 0.6 \
--top_p 0.95 \
--training_mode ${mode} \
--init_kl_coef 0.0 \
--aux_loss_coef ${aux} \
--entropy_loss_coef 0.0 \
--prompt_data $DATASET \
--input_key question \
--apply_chat_template \
--normalize_reward \
--flash_attn \
--gradient_checkpointing \
--ckpt_path $SAVE_PATH \
--save_steps 3 \
--eval_steps ${evalsteps} \
--max_ckpt_num 3 \
--save_hf_ckpt \
--disable_ds_ckpt \
--disable_fast_tokenizer \
--system_prompt ${sys} \
--use_kl_estimator_k3 \
--wandb_project vlm-rl \
--buffer_norm 0 \
--train_vlm \
--filter ${filter} \
--eval_data ${testdata} \
--data_version ${dataver} \
--loss_version ${lossver} \
--format ${fmt} \
${wandb_args[@]} \
${post_args[@]} 
```
And I notice the loss increase steeply, and mean reward decrease after 70th step, similar to the Figure 9 reported in the paper.

<img width="894" height="444" alt="Image" src="https://github.com/user-attachments/assets/57a08ed4-77f3-4aa1-9a51-4352710e2886" />
<img width="894" height="444" alt="Image" src="https://github.com/user-attachments/assets/42d0f1a6-22dd-403e-b979-063940b7f0ce" />
<img width="894" height="444" alt="Image" src="https://github.com/user-attachments/assets/45f8c149-945d-43ce-929a-991f3ae6c5cb" />

<img width="894" height="444" alt="Image" src="https://github.com/user-attachments/assets/d7e77899-e66c-4545-b790-83b79f88e14e" />

**Comment by EugeneLiu01** [comment]
By the way, how could these `post_args` be set properly in multi-node setting?
```
post_args=(--ref_num_nodes 0
    --ref_num_gpus_per_node 8 
    --actor_num_nodes ${nactor}
    --actor_num_gpus_per_node 8 
    --vllm_num_engines ${nvllm} 
    --vllm_tensor_parallel_size ${tp}
    --micro_train_batch_size ${mbsz} 
    --train_batch_size ${bsz} 
    --micro_rollout_batch_size 1
    --rollout_batch_size ${rbuffer}
)
```
